crates/web-client/src/models/account_storage.rs

```diff
@@ -22,6 +23,25 @@ impl AccountStorage {
     pub fn get_map_item(&self, index: u8, key: &Word) -> Option<Word> {
         self.0.get_map_item(index, key.into()).ok().map(Into::into)
     }
+
+    /// Returns all entries from the storage map at the given index.
+    /// Returns None if the slot is not a map or if the index is out of bounds.
```

Comment:
Just a suggestion:

```suggestion
    /// Get all key–value pairs from the map slot at `index`.
    /// Returns `undefined` if the slot isn't a map or `index` is out of bounds (0–255).
    /// Returns `[]` if the map exists but is empty.
```

We will need to improve the docs of most of the WebClient's API, so this will likely get reworked fairly soon.